modules/XMLDocuments/generators/stornoXMLGenerator.js

```diff
@@ -0,0 +1,50 @@
+import {
+  getCashboxFields,
+  getCashierFields,
+  getDateTimeFields,
+  getDoctype,
+  getDocumentNumberFields,
+  getOfflineFields,
+  getOrganizationFields,
+  getTestingModeFields,
+  getUIDFields,
+  getVersionFields,
+} from "./commonXMLTagGenerator.js";
+import { DOC_SUBTYPE_STORNO, DOC_TYPE_PRODUCT } from "../const/fiscal.js";
+
+const getStornoHeader = (operationData) => {
+  const { cashboxData, dateTime } = operationData;
+
+  return {
+    ...getTypeFields(),
+    ...getUIDFields(operationData),
+    ...getOrganizationFields(cashboxData),
+    ...getDateTimeFields(dateTime),
+    ...getDocumentNumberFields(cashboxData),
+    ...getCashboxFields(cashboxData),
+    ...getStornoDocumentNumberField(operationData),
+    ...getCashierFields(operationData),
+    ...getVersionFields(),
+    ...getOfflineFields({ operationData }),
+    ...getTestingModeFields(cashboxData),
+  };
+};
+
+const getTypeFields = () => getDoctype(DOC_TYPE_PRODUCT, DOC_SUBTYPE_STORNO);
+
+const getStornoDocumentNumberField = (data) => {
+  return {
+    ORDERSTORNUM: data.fiscalIdForStorno,
+  };
+};
```

Comment:
В нас вже є використання поля documentFiscalId. Пропоную його використовувати
```suggestion 
const getStornoDocumentNumberField = ({ documentFiscalId }) => ({
  ORDERSTORNUM: documentFiscalId,
});
```

Reply:
але це  ficalId не поточного документа, а саме документа який треба сторнувати, я тому і додав по іншому щоб не заплутати потім